stable/redmine/Chart.yaml

```diff
@@ -1,22 +1,22 @@
 apiVersion: v1
 name: redmine
-version: 13.6.0
+version: 13.7.0
```

Comment:
I would bump the major version here.

Reply:
I will support both ways to setup Ingress